monkestation/code/modules/cybernetics/implant_items/weapons/mantis_blade.dm

```diff
@@ -96,3 +82,76 @@
 		living.stamina?.adjust(-30) // cost of a lunge
 
 	attack(target, user)
+
+
+
+
+/////////SHIELD MANTIS BLADES/////////////////
+/obj/item/mantis_blade/modified
+	name = "Modified C.H.R.O.M.A.T.A. mantis blade"
+	desc = "Modified mantis blades with bigger and wider blades, allowing user to block incoming projectiles and attacks. Because of that, the edge of blades is rather dull and large which makes it worse at wounding and requires much more time between each slash."
+	icon_state = "shield_mantis"
+	inhand_icon_state = "shield_mantis"
+	lefthand_file = 'monkestation/code/modules/cybernetics/icons/swords_lefthand.dmi'
+	righthand_file = 'monkestation/code/modules/cybernetics/icons/swords_righthand.dmi'
+	force = 10
+	wound_bonus = 10
+	attack_speed = 12
+	var/in_stance = FALSE
+
+/obj/item/mantis_blade/modified/attack_self(mob/user)
```

Comment:
not gonna lie i do like the way you handle the checks here, it itches my brain quite good.